packages/flutter_nekoton_bridge/lib/nekoton/core/accounts_storage/models/account_to_add.dart

```diff
@@ -8,10 +10,10 @@ part 'account_to_add.g.dart';
 class AccountToAdd with _$AccountToAdd {
   const factory AccountToAdd({
     required String name,
-    required String publicKey,
+    @publicKeyJsonConverter required PublicKey publicKey,
     required WalletType contract,
     required int workchain,
-    String? explicitAddress,
+    @addressJsonConverter Address? explicitAddress,
```

Comment:
I think, for such cases, we must put something like `name` argument to `addressJsonConverter`, because it will try to take address from `name` field as in `address.g.dart`, but here it provides with `explicitAddress` name